src/windows/guest.c

```diff
@@ -1193,19 +1193,21 @@ void window_guest_overview_tool_update(rct_window* w, int widgetIndex, int x, in
 	y += 16;
 	RCT2_GLOBAL(RCT2_ADDRESS_PICKEDUP_PEEP_X, uint16) = x;
 	RCT2_GLOBAL(RCT2_ADDRESS_PICKEDUP_PEEP_Y, uint16) = y;
-	w->var_492++;
-	if (w->var_492 >= 48)w->var_492 = 0;
+	w->picked_peep_frame++;
+	if (w->picked_peep_frame >= 48) {
+		w->picked_peep_frame = 0;
+	}
 
 	rct_peep* peep;
 	peep = GET_PEEP(w->number);
-	int ebx = (RCT2_ADDRESS(0x982708, uint32*)[peep->sprite_type * 2])[22];
-	ebx += w->var_492 >> 2;
+	uint32 sprite_idx = (RCT2_ADDRESS(0x982708, uint32*)[peep->sprite_type * 2])[22];
```

Comment:
For readying drawing images such as this I prefer the name of `imageId` rather than `spriteId`. The global address should probably be changed to `PICKEDUP_PEEP_IMAGE` as it is an imageId as well.